osu.Game/Tests/Beatmaps/TestWorkingBeatmap.cs

```diff
@@ -50,28 +50,32 @@ public TestWorkingBeatmap(IBeatmap beatmap, IFrameBasedClock referenceClock = nu
         public class TrackVirtualManual : Track
         {
             private readonly IFrameBasedClock referenceClock;
-            private readonly ManualClock clock;
+
+            private readonly ManualClock clock = new ManualClock();
 
             private bool running;
```

Comment:
Intended for this to be false by default?

Reply:
seems correct, referencing `TrackVirtual`